api/src/main/java/io/opentelemetry/api/metrics/DefaultMeter.java

```diff
@@ -14,23 +14,18 @@
 
 /** No-op implementations of {@link Meter}. */
 @ThreadSafe
-public final class DefaultMeter implements Meter {
+final class DefaultMeter implements Meter {
 
-  private static final DefaultMeter INSTANCE = new DefaultMeter();
+  private static final DefaultMeter instance = new DefaultMeter();
```

Comment:
why lowercase? Isn't the DefaultMeter immutable?

Reply:
It seemed slightly more cool than a boring constant but that's true :)